app/models/patient.rb

```diff
@@ -48,4 +60,25 @@ def current_age
       age + years_since_update
     end
   end
+
+  def self.not_contacted
+    where(contacted_by_counsellor: false)
+      .where(could_not_contact_reason: nil)
+      .where('device_created_at <= ?', 2.days.ago)
+  end
+
+  def call_result=(new_call_result)
```

Comment:
This should be a seperate setter method. Expecting other attributes to change when assigning a value using `=` is very unintuitive.